content/en/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -198,3 +197,20 @@ Both methods remove the Agent, but they do not remove the `C:\ProgramData\Datado
 
 {{% /tab %}}
 {{< /tabs >}}
+
+### Ensure machine doesn't restart
+
+Some Agent versions may cause a forced reboot.
+To prevent this, use either `REBOOT=ReallySuppress` or `/norestart`. Both achieve the same effect.
+
+The following is an example of the `/norestart` flag.
+
+```powershell
+start-process msiexec -Wait -ArgumentList ('/log', 'C:\\uninst.log', '/norestart', '/q', '/x', (Get-WmiObject -Class Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber)
+```
+
+The following is an example of the `REBOOT=ReallySuppress` flag.
+
+```powershell
+start-process msiexec -Wait -ArgumentList ('/log', 'C:\\uninst.log', '/q', '/x', (Get-WmiObject -Class Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber, 'REBOOT=ReallySuppress')
+```
```

Comment:
We are giving three options to the customer: one using `Get-WmiObject` and two using `msiexec`, forcing them to decide which one is best for them/try them all. I think we can and should decide which is the best one for them and only recommend that one. If I'm not mistaken, `msiexec` is preferable over `Get-WmiObject` (among other things because it let's you specify the no-reboot param), and if both `msiexec` options do the same, let's give just one.